test/performance/latency/LatencyTestSubscriber.hpp

```diff
@@ -33,6 +33,14 @@
 #include <fastrtps/types/DynamicData.h>
 #include <fastrtps/types/DynamicPubSubType.h>
 
+#include <fastdds/dds/topic/TypeSupport.hpp>
```

Comment:
Include order.

Reply:
Done in 82bfe8959317b5e4404ad4d5bce0eac19c2ebef7